src/globals.css

```
@@ -133,6 +142,66 @@
+  /* 탭 페이지가 하단 플로팅 탭바에 가리지 않도록 주는 여백.
+   *
+   * ⚠️ 반드시 **배경색을 가진 스크롤 컨테이너 자신**에 붙일 것.
+   *    상위 래퍼(TabBarLayout 의 main 등)에 padding 으로 주면 그 영역이 자식 배경 박스
+   *    바깥이라, 페이지 배경이 화면 아래끝까지 못 가고 MainLayout 의 흰색이 드러난다.
+   *    그 경계선이 스크롤할 때 따라다니는 것처럼 보인다(일정 목록 탭에서 실제로 발생). */
+  /* 기준은 탭바가 아니라 **FAB** 다. FAB 상단이 화면 하단에서 160px(104+56)라
+   * 탭바(92px)만 피하면 마지막 카드가 FAB 에 깔린다. 여기에 숨 쉴 틈 16px 를 더한다.
+   * = 176px. 스크림(200px)의 상단 12% 구간이라 알파가 사실상 0 이어서 흐려지지도 않는다. */
+  .pb-tabbar {
+    padding-bottom: calc(
+      var(--fab-bottom) + var(--fab-size) + 16px +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)